cmd/kubeadm/app/cmd/token.go

```diff
@@ -221,7 +221,19 @@ func RunCreateToken(out io.Writer, client clientset.Interface, token string, tok
 		}
 	}
 
-	// TODO: Validate usages here so we don't allow something unsupported
+	invalidUsages := sets.NewString()
```

Comment:
I want _this_ code to be in `pkg/bootstrap/api`; the code that supports multiple usages, not only one.